packages/server/database/migrations/20210307124052-prepareBackfillTeams.ts

```diff
@@ -0,0 +1,33 @@
+export const up = async function (r) {
+  try {
+    await Promise.all([
+      r.db('actionProduction')
+        .table('Team')
+        .indexCreate('createdAt')
```

Comment:
are these indices purely for the postMigration?
Might be worth just doing `updatedAt` and then setting updatedAt === createdAt if it's null.
that way we only need to reference updatedAt

Reply:
i love this. also changed in user migration PR